src/palette/internal/palette/palettecreator.cpp

```diff
@@ -1099,11 +1099,15 @@ PalettePanel* PaletteCreator::newBracketsPalettePanel()
             { BracketType::LINE,   QT_TRANSLATE_NOOP("palette", "Line") } }
     };
 
-    for (auto type : types) {
+    static Staff bracketItemOwner(nullptr);
+    bracketItemOwner.setBracketType(types.size() - 1, BracketType::NORMAL);
```

Comment:
this seems to be the correct way to ensure creation/deletion of BracketItem's is handled entirely by their parent (Staff).